include/deal.II/base/utilities.h

```diff
@@ -389,10 +389,25 @@ namespace Utilities
      */
     struct MemoryStats
     {
-      unsigned long int VmPeak; /** peak virtual memory size in kB */
-      unsigned long int VmSize; /** current virtual memory size in kB */
-      unsigned long int VmHWM; /** peak resident memory size in kB */
-      unsigned long int VmRSS; /** current resident memory size in kB */
+      /**
+       * peak virtual memory size in kB
```

Comment:
While you're already doing this, would you mind capitalizing the first word and putting a period at the end of the sentence for each of these entries?